### ISSUE TITLE
Portins Order Can Contain `ActualFocDate`

### ISSUE THREAD

**Issue by dtolb** [Bandwidth/php-bandwidth-iris#23]
The Portins model is missing the `ActualFocDate` on `GET` `/accounts /{accountId} /portins /{orderid}`

`   <ActualFocDate>2018-02-23T16:30:00Z</ActualFocDate>`

```xml
<?xml version="1.0" encoding="UTF-8"?>
<LnpOrderResponse>
   <ProcessingStatus>FOC</ProcessingStatus>
   <RequestedFocDate>2018-02-23T16:30:00Z</RequestedFocDate>
   <ActualFocDate>2018-02-23T16:30:00Z</ActualFocDate>
   <LoaAuthorizingPerson>loa name</LoaAuthorizingPerson>
   <Subscriber>
      <SubscriberType>BUSINESS</SubscriberType>
      <BusinessName>Firs Last</BusinessName>
      <ServiceAddress>
         <HouseNumber>21</HouseNumber>
         <StreetName>E st</StreetName>
         <City>Tres</City>
         <StateCode>AK</StateCode>
         <Zip>25598</Zip>
         <Country>United States</Country>
         <AddressType>Service</AddressType>
      </ServiceAddress>
   </Subscriber>
   <BillingTelephoneNumber>9192000161</BillingTelephoneNumber>
   <ListOfPhoneNumbers>
      <PhoneNumber>9192000161</PhoneNumber>
   </ListOfPhoneNumbers>
   <PON>68120b50-4903-44b4-829d-39fa3bd878a1</PON>
   <AccountId>6555555</AccountId>
   <SiteId>3</SiteId>
   <PeerId>516144</PeerId>
   <LosingCarrierName>Bandwidth</LosingCarrierName>
   <VendorName>Bandwidth CLEC</VendorName>
   <OrderCreateDate>2018-02-20T15:17:21.649Z</OrderCreateDate>
   <LastModifiedDate>2018-02-20T15:17:32.569Z</LastModifiedDate>
   <userId>System</userId>
   <LastModifiedBy>System</LastModifiedBy>
   <PartialPort>true</PartialPort>
   <Immediately>false</Immediately>
   <Triggered>false</Triggered>
   <BillingType>NOCHARGE</BillingType>
   <PortType>INTERNAL</PortType>
   <OverrideValidation>false</OverrideValidation>
</LnpOrderResponse>
```

**Comment by ooglek** [comment]
I'll submit a pull request if this can happyen!

**Comment by ooglek** [comment]
This can be closed as fixed. It was done and released in v 2.0.1 by @jmulford-bw on July 26, 2019.

https://github.com/Bandwidth/php-bandwidth-iris/commit/f16a130b2d171eff66898b87ccdb3da88d20d58b
https://github.com/Bandwidth/php-bandwidth-iris/releases/tag/v2.0.1
